File: examples/js/get__intelligence_statistics.js

const authToken = process.env.AIOPS_TOKEN || 'YOUR_TOKEN';
// 示例：GET /intelligence/statistics
fetch(`http://localhost:8000/api/v1/intelligence/statistics`, {
    method: 'GET',
    headers: { Authorization: `Bearer ${authToken}` }
})
.then(res => res.json())
.then(data => console.log(data))
.catch(err => console.error('Request error:', err));

File: examples/python/get__intelligence_statistics.py
# -*- coding: utf-8 -*-
import requests

# 示例：GET /intelligence/statistics
# 使用 GET 方法请求 /intelligence/statistics
url = "http://localhost:8000/api/v1/intelligence/statistics"
headers = {"Authorization": "Bearer YOUR_TOKEN"}

try:
    response = requests.get(url, headers=headers)
    print("Status:", response.status_code)
    print("Response:", response.json())
except Exception as e:
    print("Request failed:", e)
